functions/src/utils/agora.js

```diff
@@ -0,0 +1,41 @@
+const functions = require("firebase-functions");
+
+const { RtcTokenBuilder, RtcRole } = require("agora-access-token");
+
+const AGORA_CONFIG = functions.config().agora;
+
+const appId = AGORA_CONFIG.app_id;
+const appCertificate = AGORA_CONFIG.app_certificate;
+
+// @debt should we move this into AGORA_CONFIG, or maybe venue config in firestore or similar?
+const expirationTimeInSeconds = 3600;
```

Comment:
> should we move this into AGORA_CONFIG, or maybe venue config in firestore or similar?

[note] Open to thoughts/suggestions on what we do here. For this PR it's probably good enough as is. Need to read into it a bit more to see what happens when they expire, how renewal is handled, and if it will disrupt an ongoing video call if it expires part way through. So probably OOS to change in this PR.